wsmaster/che-core-sql-schema/src/main/resources/che-schema/6.4.0/postgresql/2__add_signature_key.sql

```diff
@@ -1,5 +1,5 @@
 --
--- Copyright (c) 2012-2017 Red Hat, Inc.
+-- Copyright (c) 2012-2019 Red Hat, Inc.
```

Comment:
Please revert changes in sql scripts because it will break all existing Che Installation